tutorials/basic/mnist.py

```diff
@@ -58,7 +60,7 @@ def main(_):
   print(
       f'Training model with learning rate={FLAGS.learning_rate} for {FLAGS.epochs} epochs.'
   )
-  model.fit(x_train, y_train, epochs=FLAGS.epochs, verbose=2)
+  model.fit(x_train, y_train, epochs=FLAGS.epochs)
```

Comment:
now we can allow the progress bars to shine through!

Reply:
vary nice